app/models/manageiq/providers/vmware/infra_manager/provision_workflow.rb

```diff
@@ -28,83 +28,83 @@ def allowed_provision_types(_options = {})
+    '045' => "(GMT-05:00) Bogota, Lima, Quito",
+    '050' => "(GMT-04:00) Atlantic Time (Canada)",
+    '055' => "(GMT-04:00) Caracas, La Paz",
+    '056' => "(GMT-04:00) Santiago",
+    '060' => "(GMT-03:30) Newfoundland and Labrador",
+    '065' => "(GMT-03:00) Brasilia",
+    '070' => "(GMT-03:00) Buenos Aires, Georgetown",
+    '073' => "(GMT-03:00) Greenland",
+    '075' => "(GMT-02:00) Mid-Atlantic",
+    '080' => "(GMT-01:00) Azores",
+    '083' => "(GMT-01:00) Cape Verde Islands",
+    '085' => "(GMT-00:00) Greenwich Mean Time : Dublin, Edinburgh, Lisbon, London",
+    '090' => "(GMT-00:00) Casablanca, Monrovia",
+    '095' => "(GMT+01:00) Belgrade, Bratislava, Budapest, Ljubljana, Prague",
+    '100' => "(GMT+01:00) Sarajevo, Skopje, Warsaw, Zagreb",
+    '105' => "(GMT+01:00) Brussels, Copenhagen, Madrid, Paris",
+    '110' => "(GMT+01:00) Amsterdam, Berlin, Bern, Rome, Stockholm, Vienna",
+    '113' => "(GMT+01:00) West Central Africa",
+    '115' => "(GMT+02:00) Bucharest",
+    '120' => "(GMT+02:00) Cairo",
+    '125' => "(GMT+02:00) Helsinki, Kiev, Riga, Sofia, Tallinn, Vilnius",
+    '130' => "(GMT+02:00) Athens, Istanbul, Minsk",
+    '135' => "(GMT+02:00) Jerusalem",
+    '140' => "(GMT+02:00) Harare, Pretoria",
+    '145' => "(GMT+03:00) Moscow, St. Petersburg, Volgograd",
+    '150' => "(GMT+03:00) Kuwait, Riyadh",
+    '155' => "(GMT+03:00) Nairobi",
+    '158' => "(GMT+03:00) Baghdad",
+    '160' => "(GMT+03:30) Tehran",
+    '165' => "(GMT+04:00) Abu Dhabi, Muscat",
+    '170' => "(GMT+04:00) Baku, Tbilisi, Yerevan",
+    '175' => "(GMT+04:30) Kabul",
+    '180' => "(GMT+05:00) Ekaterinburg",
+    '185' => "(GMT+05:00) Islamabad, Karachi, Tashkent",
+    '190' => "(GMT+05:30) Chennai, Kolkata, Mumbai, New Delhi",
+    '193' => "(GMT+05:45) Kathmandu",
+    '195' => "(GMT+06:00) Astana, Dhaka",
+    '200' => "(GMT+06:00) Sri Jayawardenepura",
+    '201' => "(GMT+06:00) Almaty, Novosibirsk",
+    '203' => "(GMT+06:30) Yangon (Rangoon)",
+    '205' => "(GMT+07:00) Bangkok, Hanoi, Jakarta",
+    '207' => "(GMT+07:00) Krasnoyarsk",
+    '210' => "(GMT+08:00) Beijing, Chongqing, Hong Kong SAR, Urumqi",
+    '215' => "(GMT+08:00) Kuala Lumpur, Singapore",
+    '220' => "(GMT+08:00) Taipei",
+    '225' => "(GMT+08:00) Perth",
+    '227' => "(GMT+08:00) Irkutsk, Ulaanbaatar",
+    '230' => "(GMT+09:00) Seoul",
+    '235' => "(GMT+09:00) Osaka, Sapporo, Tokyo",
+    '240' => "(GMT+09:00) Yakutsk",
+    '245' => "(GMT+09:30) Darwin",
+    '250' => "(GMT+09:30) Adelaide",
+    '255' => "(GMT+10:00) Canberra, Melbourne, Sydney",
+    '260' => "(GMT+10:00) Brisbane",
+    '265' => "(GMT+10:00) Hobart",
+    '270' => "(GMT+10:00) Vladivostok",
+    '275' => "(GMT+10:00) Guam, Port Moresby",
+    '280' => "(GMT+11:00) Magadan, Solomon Islands, New Caledonia",
+    '285' => "(GMT+12:00) Fiji Islands, Kamchatka, Marshall Islands",
+    '290' => "(GMT+12:00) Auckland, Wellington",
+    '300' => "(GMT+13:00) Nuku'alofa"
+  }.freeze
```

Comment:
And while you are at it single quotes would work on the value side of the hash.  Note `"(GMT+13:00) Nuku'alofa"` requires escaping: `'(GMT+13:00) Nuku\'alofa'`